sdk/js/src/EventEmitter.ts

```diff
@@ -94,12 +96,17 @@ export class EventEmitter {
         })
     }
 
-    emitFeatureUpdates(oldFeatureSet: DVCFeatureSet, newFeatureSet: DVCFeatureSet): void {
-        const keys = Object.keys(oldFeatureSet).concat(Object.keys(newFeatureSet))
+    emitFeatureUpdates(
+        oldFeatureSet: DVCFeatureSet | undefined, 
```

Comment:
how come this needs to take undefined? Isn't an empty object a valid DVCFeatureSet?

Reply:
I made the change since if the old config isn't defined yet it will be defaulted to `{}` which will make pulling the features/variables undefined, but I can make it be defined higher up so i don't have to change the signature